spec.bs

```diff
@@ -320,6 +336,19 @@ When {{Worklet/addModule()}} is called for a worklet, it will run [=check if mod
     1. [=map/Set=] the value of |operationMap|[|name|] to |run|.
   </div>
 
+  <div algorithm>
+    The <dfn for="SharedStorageWorkletGlobalScope">sharedStorage getter</dfn> steps are:
+
+    1. If |this|'s [=addModule success=] is true, return |this|'s {{SharedStorageWorkletGlobalScope/sharedStorage}}.
+    1. Otherwise, return null.
```

Comment:
I don't think matches the current implementation. When this is accessed, I think there's instead a DOMException with name NotAllowedError.

Reply:
Good point. 

I put a TypeError in the spec for consistency with the rest of the spec (and per a previous comment from wanderview@).

Reply:
Sg! Defer to you and wanderview@, but I'm also wondering two more things: should this return undefined instead of null? And is `sharedStorage` still default initialized if its type has the `?` optional marker?

Reply:
Ah actually -- just realized that throwing terminates execution (https://infra.spec.whatwg.org/#algorithm-control-flow), so I don't think you need to return null (and so can probably remove the `?` optional marker, I think)

Reply:
Done, thanks.